core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -82,22 +98,35 @@ oppia.controller('Dashboard', [
       } else {
         $scope.currentSortType = sortType;
       }
-      $scope.explorationsList = (sortExplorationsService.sortBy(
-        $scope.explorationsList,
-        $scope.currentSortType,
-        $scope.isCurrentSortDescending));
+    };
+
+    $scope.sort = function(entity) {
```

Comment:
I recommend a clearer name, like `sortByFunction` and a comment explaining what this function is used for.